lib/datadog/tracing/contrib/pg/instrumentation.rb

```diff
@@ -0,0 +1,75 @@
+# typed: false
+
+require 'datadog/tracing'
+require 'datadog/tracing/metadata/ext'
+require 'datadog/tracing/contrib/analytics'
+require 'datadog/tracing/contrib/pg/ext'
+
+module Datadog
+  module Tracing
+    module Contrib
+      module Pg
+        # PG::Connection patch module
+        module Instrumentation
+          def self.included(base)
+            base.prepend(InstanceMethods)
+          end
+
+          # PG::Connection patch methods
+          module InstanceMethods
+            def exec(sql)
+              service = Datadog.configuration_for(self, :service_name) || datadog_configuration[:service_name]
+
+              Tracing.trace(Ext::SPAN_EXEC, service: service) do |span|
+                span.resource = sql
+                span.type = Tracing::Metadata::Ext::SQL::TYPE
+                span.service = Ext::DEFAULT_PEER_SERVICE_NAME
+
+                span.set_tag('db.instance', db)
+                span.set_tag('db.user', user)
+                span.set_tag('db.system', Ext::SPAN_SYSTEM)
+                span.set_tag('db.statement', sql)
```

Comment:
Correct me if I'm wrong here, but these are tags that Datadog would like to set in all database integrations, right?

If so, let's treat them a shared resource and move them to a shared module.

I suggest we move these to `lib/datadog/tracing/metadata/ext.rb`, and I recommend creating a new `module` there inside `Ext` called `Db`.

The tags will look like this in the `ext.rb` file: `TAG_DB_USER = 'db.user'`